server-spi-private/src/main/java/org/keycloak/services/resource/AccountResourceSpi.java

```diff
@@ -0,0 +1,34 @@
+package org.keycloak.services.resource;
+
+import org.keycloak.provider.Provider;
+import org.keycloak.provider.ProviderFactory;
+import org.keycloak.provider.Spi;
+
+/**
+ * <p>A {@link Spi} to replace Account resources.
+ *
+ * <p>Implementors can use this {@link Spi} to override the behavior of the Account endpoints and resources by
+ * creating JAX-RS resources that override those served at /account by default.
+ */
+public class AccountResourceSpi implements Spi {
+
+    @Override
+    public boolean isInternal() {
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "account-console";
```

Comment:
Perhaps `account-resource` to make more explicit it is related to the backend and resources are served? Also aligned with the usages of `Resource` in javadocs and type names.

Reply:
changed name to `account-resource`